frmts/georaster/georasterdrivercore.cpp

```diff
@@ -100,6 +100,33 @@ void GEORDriverSetCommonMetadata(GDALDriver *poDriver)
         "  </Option>"
         "  <Option name='GENPYRLEVELS'  type='int'  description='Number of "
         "pyramid level to generate'/>"
+        " <Option name='GENSTATS' type='boolean'"
+        "description='Generate statistics from the given rasters'"
```

Comment:
I suspect missing spaces cause XML validation issues:
```suggestion
        "description='Generate statistics from the given rasters' "
```

Reply:
You're right, I might missed those when merging with the latest main changes, I'm going to make a commit with those fixes.